src/ventus/wm/modes/expose.js

```diff
@@ -9,25 +9,34 @@ define([
 ], function(_, Promise) {
 	'use strict';
 
+	var exposeOnRightClick = true;
+
 	var ExposeMode = {
+		// Enable/disable expose on right-click (true=disable / false=enable)
+		setExposeOnRightClick: function (value) {
```

Comment:
To keep consistency with other parts of the code, we could use a setter for this.